torch/csrc/jit/passes/quantization.cpp

```diff
@@ -164,11 +191,27 @@ void InsertObserverNodes(std::shared_ptr<Graph>& graph, Node* observer_node) {
   // Actually add observer nodes.
   for (Value* v : values_to_observe) {
     if (v->type()->isSubtypeOf(TensorType::get())) {
-      addObserverFor(v, observer_node);
+      Node* clone_observer_node = addObserverFor(v, observer_node, v->node());
+      clone_observer_node->insertAfter(v->node());
     }
   }
 }
 
+void InsertObserverNodes(
+    std::shared_ptr<script::Module>& moduleObj,
+    const std::string& method_name,
+    Node* observer_node) {
+  const auto& method = moduleObj->get_method(method_name);
+  InsertObserverNodes(method.graph(), observer_node, method.num_inputs());
+}
+
+void InsertObserverNodes(
```

Comment:
and this as well. since we already have overload in `_jit_pass_insert_observers`